setup-signify.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -euo pipefail 
+
+sudo apt -y install signify-openbsd qrencode
+
+# clear out old keys
+rm -f key.pub key.sec
+
+# Generate the keypair
+signify-openbsd -G -n -p /vx/config/key.pub -s /vx/config/key.sec
+
+# Output the public key for enrollment into another device
+qrencode -t UTF8 -r key.pub -o -
```

Comment:
same here, should this be `/vx/config/key.pub` ?

Reply:
Fixed